src/features/expenses/ChangeUserMoneyModal/index.tsx

```diff
@@ -30,8 +26,21 @@ const ChangeUserMoneyModal = ({ user, onDone, close, amount }: IProps) => {
     amount: amount > 0 ? amount.toString() : '',
   });
 
+  const sanitize = (value: string): string => {
+    let sanitizedValue = value.replace(',', '.');
+        
+    if (sanitizedValue.includes('.')) {
+      if (+(+sanitizedValue).toFixed(2) === +sanitizedValue)
+        return sanitizedValue;
+
+      return sanitizedValue.substring(0, sanitizedValue.length - 1);
+    }
+    
+    return sanitizedValue;
+  }
+
```

Comment:
Move it outside of the component
It doesn't depend on component entrails, so we don't need to recreate it on each component rerender